financial-connections/src/main/java/com/stripe/android/financialconnections/di/FinancialConnectionsSheetComponent.kt

```diff
@@ -21,15 +21,15 @@ import javax.inject.Singleton
 internal interface FinancialConnectionsSheetComponent {
     val viewModel: FinancialConnectionsSheetViewModel
 
-    fun inject(factory: FinancialConnectionsSheetViewModel.Factory)
+    fun inject(factory: FinancialConnectionsSheetViewModel.Companion)
```

Comment:
The reason we have the inject method here is to ensure the factory was injectable when a fallback component injection is created, it needs to be explicitly called(e.g [here](https://github.com/stripe/stripe-android/blob/6ba4a285c00fcb1127125f85cee405e2dc3a21a5/identity/src/main/java/com/stripe/android/identity/viewmodel/IdentityViewModel.kt#L568)), since fallback is not yet implemented, we could remove this method

Reply:
Forgot to remove it https://github.com/stripe/stripe-android/pull/4986/commits/e5e6fbc5e797c24177849b03249b14e374c13466